src/parameters/parameters_ode_solver.cpp

```diff
@@ -22,6 +22,25 @@ void ODESolverParam::declare_parameters (dealii::ParameterHandler &prm)
                           dealii::Patterns::Double(0,dealii::Patterns::Double::max_double_value),
                           "Outputs the solution at time intervals of dt in .vtk file");
 
+        prm.declare_entry("output_solution_at_fixed_times", "false",
+                          dealii::Patterns::Bool(),
+                          "Output solution at fixed times. False by default.");
+
+        prm.declare_entry("output_solution_fixed_times_string", " ",
+                          dealii::Patterns::FileName(dealii::Patterns::FileName::FileType::input),
+                          "String of the times at which to output the velocity field. "
+                          "Example: '0.0 1.0 2.0 3.0 '");
+
+        prm.declare_entry("number_of_fixed_times_to_output_solution", "0",
```

Comment:
I would make the code measure the length of `output_solution_fixed_times_string` rather than asking the user to input it. I imagine it would be easy to miscount.

Reply:
fixed in most recent commit

Reply:
Thanks.